cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,83 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+// eslint-disable-next-line max-len
+import { generateArticle, generateNewArticle } from '../support/generateArticle';
 
-  });
+const user = {
+  username: 'wombat098',
+  email: 'wombat098@i.ua',
+  password: '1234567890'
+};
+
+const { title, description, body, tag } = generateArticle();
+
+const { newTitle, newDescription, newBody, newTag } = generateNewArticle();
 
+const articlePayload = {
+  title,
+  description,
+  body,
+  taglist: [
+    tag
+  ]
+};
+
+describe('Article', () => {
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.login(user);
```

Comment:
Ensure that the `cy.login(user)` command is correctly implemented in your support files. This command should handle the login process for the user object provided.